main/integration-tests/__test__/api-tests/projects/get-projects.test.js

```diff
@@ -0,0 +1,59 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License").
+ *  You may not use this file except in compliance with the License.
+ *  A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+const { runSetup } = require('../../../support/setup');
+const errorCode = require('../../../support/utils/error-code');
+
+describe('List projects scenarios', () => {
+  let setup;
+  let adminSession;
+
+  beforeAll(async () => {
+    setup = await runSetup();
+    adminSession = await setup.defaultAdminSession();
+  });
+
+  afterAll(async () => {
+    await setup.cleanup();
+  });
+
+  describe('List projects', () => {
+    it('should fail if user is inactive', async () => {
+      const admin2Session = await setup.createAdminSession();
+      await adminSession.resources.users.deactivateUser(admin2Session.user);
+
+      await expect(admin2Session.resources.projects.get()).rejects.toMatchObject({
+        code: errorCode.http.code.unauthorized,
+      });
+    });
+
+    it('should pass if internal guest attempts to get project list', async () => {
```

Comment:
BTW,  Eyor used to have the pattern 'should pass',  but then to be consistent,  I suggested that we rephrase it.
So in your case, you can say 'should return project list for internal guest'.   For the case of 'should fail', you can leave it the way it is.  At  least that is what I have seen we are doing in the previous tests, I thought I mentioned this so that we be consistent in the description.